Assets/MixedRealityToolkit/_Core/Definitions/Utilities/SortType.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities
+{
+    /// <summary>
+    /// Sorting type for collections
+    /// </summary>
+    public enum SortType
```

Comment:
Does vNext end enum names with "Enum"? i.e, SortTypeEnum. I've seen both in htk (SortType and SortTypeEnum), but I'm not sure what the guideline is for vNext.

Reply:
Type is the suffix we've been going with. But I'm fine with either as long as we make a decision.

Reply:
Thats what I thought too, but going through everything, I didn't find any enum suffixes so i've since renamed everything. Personally, I feel having the suffix helps with self-documentation.

Reply:
I think this now has both type suffixes XD